DietPi-Software | Navidrome

Joulinar opened the pull request:
- Check if `navidrome.toml` exists before creating it.

Joulinar committed:
v8.11

DietPi-Software | Navidrome - check if navidrome.toml exists before creating it

MichaIng reviewed (dismissed):
Let's add a changelog entry as well.

MichaIng committed:
v8.11

- CHANGELOG | navidome: Resolved an issue where on reinstalls the config was was overwritten. Many thanks to @pedrom34 for reporting this issue: https://github.com/MichaIng/DietPi/issues/5840
